README.md

```diff
@@ -168,3 +168,23 @@ Additionally, response can contain additional response body. All fields of the r
   "payment": {"id":"XXX", "intent":"sale", ...}  # only in case of getting the payment object 
 }
 ```
+
+## Possible edge cases
+1. First case:
+    1. user inputs an address Berlin, Germany
+    1. he clicks on Continue and is redirected to Paypal payment page with this address.
+    1. In a different tab, the same user changes his address to Paris, France.
+    1. The user confirms the payment in the first tab
+    1. Paypal has the address of Berlin, but the shop will deliver the goods to Paris.
+    
+    **Possible solution:** the backend calls `patch/payments` endpoint every time user changes it.
```

Comment:
- _backend_ or _frontend_ calls patch?

Reply:
Should be backend:
```
* The frontend -  the browser part of the shop. This is what the user interacts with.
* The backend - the shop server.
```

Reply:
yes, but how backend get known the a changes the address? Only front-end could know this and call the patch, isn't it?